### PR TITLE
Form cart summary

### PR DESCRIPTION
**Pull request by nexovocni**
I finished summary component on Form page.

### CODE REVIEW

File: src/components/SummaryTax/SummaryTax.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+                    itemsValue="TBD"
+                    disabledCode={disabledCode} 
+                    title="Taxes"
+                />
+            </div>
+            <div className={tax ? "summary-first-total-open" : "summary-first-total"} >
+                <SummaryTotal 
+                    disabledCode={disabledCode} 
+                    title="GST"
+                    itemsValue="$3.01"
```
Comment (tomapaunovic) [inline src/components/SummaryTax/SummaryTax.tsx]: Get this from API

File: src/components/SummaryTax/SummaryTax.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+                />
+                <SummaryTotal 
+                    disabledCode={disabledCode} 
+                    title="PST"
+                    itemsValue="$1.99"
```
Comment (tomapaunovic) [inline src/components/SummaryTax/SummaryTax.tsx]: Get this from API

File: src/components/SummaryTotal/SummaryTotal.tsx [hunk 3]
```diff
@@ -1,19 +1,20 @@
 import React from 'react'
-import './SummaryData.scss'
+import './SummaryTotal.scss'
 
 interface IProps {
     disabledCode: boolean,
-    itemsValue: number,
+    itemsValue: any,
```
Comment (tomapaunovic) [inline src/components/SummaryTotal/SummaryTotal.tsx]: why any? better put number | string, or something like that

File: src/components/code/code.tsx [hunk 4]
```diff
@@ -4,7 +4,7 @@ import './Code.scss'
 interface IProps {
     code: boolean
     setCode: any
-    stateComponent: boolean
+    stateComponent: any
```
Comment (tomapaunovic) [inline src/components/code/code.tsx]: Avoid using 'any'

File: src/components/SummaryTotal/SummaryTotal.tsx (hunk 3 above)
Comment (nexovocni) [reply src/components/SummaryTotal/SummaryTotal.tsx]: I put any just in order to get code work properly, now I chahged it to number and it works just fine.

File: src/components/code/code.tsx (hunk 4 above)
Comment (nexovocni) [reply src/components/code/code.tsx]: I put any just in order to get code work properly, now I chahged it to boolean and it works just fine.

File: src/components/SummaryTax/SummaryTax.tsx (hunk 1 above)
Comment (nexovocni) [reply src/components/SummaryTax/SummaryTax.tsx]: I put in a constant.

File: src/components/SummaryTax/SummaryTax.tsx (hunk 2 above)
Comment (nexovocni) [reply src/components/SummaryTax/SummaryTax.tsx]: I put in a constant.

**Pull request merged**